setup.py

```diff
@@ -99,6 +99,10 @@
     'Topic :: Security',
     'Topic :: Software Development'
   ],
+  project_urls = {
+    'Source': 'https://github.com/secure-systems-lab/securesystemslib',
+    'Issues': 'https://github.com/secure-systems-lab/securesystemslib/issues',
+  },
```

Comment:
This is a change I might run by test.pypi.org (just added you as a collaborator).

Reply:
A similar change in [tuf](https://github.com/theupdateframework/tuf/commit/eb1c8d0845bb86eb6d47da2734471fe3fadf2155) looks pretty nice. See Project Links on the left side of https://pypi.org/project/tuf/